### ISSUE TITLE
Create a Trustee dashboard page widget skeleton

### ISSUE THREAD

**Issue by ailisp** [NEAR-DevHub/neardevhub-bos#621]
**Problem**
Currently the trustee need to manually check proposals and manually issue the payment

**User Story**
As a trustee, they can login with the trustee account and see proposals to approve (to pay), payment history, and statistics of treasury

**Acceptance Criteria**
- [ ] create an empty page with a card saying only trustees can access this dashboard, and ask user to login the trustee account
- [ ] Implement the "Need Approvals" panel under "Trustees" tab, with stub data
- [ ] Implement the "Payment history" panel under "Trustees" tab, with stub data
- [ ] Implement the "Treasury" panel under "Trustees" tab, with stub data 
- [ ] The panel style and layouts should match [the figma design](https://www.figma.com/file/Tf8FIz3jKZDYZqUfc4iiQN/Q4-2023-DevHub-Platform%3A-Feature-Exploration?type=design&node-id=1-1465&mode=design&t=u9kIJnsq3lhi0Kwf-0)

**Out of Scope**
- In this ticket, we only implement the widget UI according to the design without actually integrating with the contract. The contract need some research that can be done in parallel with this task.
- The login screen in the design is also not included. For now we rely on the gateway's login feature. For a trustee account they must logout their personal account then login with their trustee account. This may change in a future ticket.
- The "Moderators" tab in the design and any content under it is not part of this ticket.

**Comment by Megha-Dev-19** [comment]
hey @frol, should we move the trustees related stories to the other repo?
